war/src/main/js/tooltips.js

```diff
@@ -0,0 +1,81 @@
+import tippy from "tippy.js"
+
+const TOOLTIP_BASE = {
+  arrow: false,
+  theme: "tooltip",
+  animation: "tooltip",
+  appendTo: document.body
+}
+
+let tooltipInstances = []
+const globalPlugin = {
+  fn() {
+    return {
+      onCreate(instance) {
+        tooltipInstances = tooltipInstances.concat(instance);
+      },
+      onDestroy(instance) {
+        tooltipInstances = tooltipInstances.filter(i => i !== instance);
+      }
+    }
+  }
+}
+
+tippy.setDefaultProps({
+  plugins: [globalPlugin]
+})
+
+registerTooltips()
+
+/**
+ * Registers tooltips for the page
+ * If called again, destroys existing tooltips and registers them again (useful for progressive rendering)
+ * @param {HTMLElement} container - Registers the tooltips for the given container
+ */
+function registerTooltips(container = document) {
+  tooltipInstances.forEach(instance => {
+    if (instance.props.container === container) {
+      instance.destroy()
+    }
+  })
+
+  tippy(container.querySelectorAll("[tooltip]:not([tooltip=\"\"])"), {
+    content: element => element.getAttribute("tooltip")
+      .replace("<br>", "\n")
+      .replace("<br/>", "\n")
+      .replace("\\n", "\n"),
```

Comment:
```suggestion
      .replace("<br>", "\n")
      .replace("<br/>", "\n")
      .replace("<br />", "\n")
      .replace("\\n", "\n"),
```
We have various occurrences of this type too.